src/main/java/nl/hsac/fitnesse/fixture/slim/JsonHttpTest.java

```diff
@@ -66,6 +66,17 @@ public String allJsonPathMatches(String expr) {
         return result;
     }
 
+    /**
+     * Update a value in a the response by supplied jsonPath
+     * @param path the jsonPath to locate the key whose value needs changing
+     * @param value the new value to set
+     */
+    public void setJsonPathTo(String path, String value){
+        String jsonStr = getResponse().getResponse();
+        String jsonPath = getPathExpr(path);
+        getResponse().setResponse(pathHelper.updateJsonPathWithValue(jsonStr, jsonPath, value));
```

Comment:
Pushed.
I have a utility method for that, but it's pretty specific for this project (versions are letters, in an excel-like range A to Z, then AA to ZZ, etc.). I have it in a custom fixture class extending from jsonHttpTest in this case.

I think this kind of thing will most of the time be very project-specific and should be in an custom fixture class..